webrpc.go

```diff
@@ -29,21 +31,33 @@ func ParseSchemaFile(path string) (*schema.WebRPCSchema, error) {
 		// Use root FS to allow RIDL file imports from parent directories,
 		// ie. import ../../common.ridl.
 
-		root := "/"
-
 		// Support Windows paths. Currently only supports paths on the same volume.
-		if volume := filepath.VolumeName(absolutePath); volume != "" {
-			root = volume + "/"
-		}
+		root := filepath.VolumeName(absolutePath) + "/"
+
+		basePath, _ := os.Getwd()
 
-		path := filepath.ToSlash(absolutePath[len(root):])
+		path, err := filepath.Rel(basePath, absolutePath)
+		if err != nil {
+			return nil, fmt.Errorf("get relative path %q: %w", path, err)
+		}
 
-		rootFS := os.DirFS(root)
+		rootFS := rootFS{
+			FS:       os.DirFS(root),
+			BasePath: basePath,
+		}
 
-		r := ridl.NewParser(rootFS, path)
-		return r.Parse()
+		return ridl.NewParser(&rootFS, path).Parse()
 
 	default:
 		return nil, fmt.Errorf("invalid schema file extension %q", ext)
 	}
 }
+
+type rootFS struct {
+	fs.FS
+	BasePath string
+}
+
+func (r *rootFS) Open(path string) (fs.File, error) {
+	return r.FS.Open(strings.TrimPrefix(filepath.Join(r.BasePath, path), "/"))
+}
```

Comment:
Are these changes needed? This was perviously tested on both Linux/Macos/Windows.

Reply:
This wrapper keeps track of the current working directory, and it expects a path relative to the working directory.
When opening a file it joins with the base path and removes the initial `/` since the root filesystem is `/` or `drive:/` on Windows.

This workaround allows to generate the debug template with the filepath relative to the working directory.